html/partials/500.shtml

```diff
@@ -1,2 +1,10 @@
-<h2>500 Internal server error</h2>
-<p>There's a problem. Please try again in a few moments.</p>
+<p><strong><h1></h1>Something's gone wrong with our servers</h1></strong></p>
+<p>We're fixing this as quickly as possible.</p>
+<p>Please try again.</p>
+<p>If you're still having difficulty:</p>
```

Comment:
The content was only recently agreed with Fola and the rest of the content designers. What is the need to change it?

In particular, it seems odd that we would state that we are working to fix the problem (are we?) but also asking the user to try again.